nilearn/glm/tests/test_first_level.py

```diff
@@ -623,8 +640,8 @@ def test_fmri_inputs_shape(tmp_path, shape_4d_default):
     )
 
 
-def test_fmri_inputs_design_matrices_tsv(tmp_path, shape_4d_default):
-    # Test processing of FMRI inputs
+def test_fmri_inputs_design_matrices_csv(tmp_path, shape_4d_default):
+    """Test design_matrices can be passed as csv."""
     mask, func_img, des = write_fake_fmri_data_and_design(
```

Comment:
Same as above, did you leave `write_fake_fmri_data_and_design` in purpose?

Reply:
yeah there are a few cases where the test actually checks if things can run with files from disk so I left it there in those cases